pkgs/top-level/python-packages.nix

```diff
@@ -26314,6 +26314,25 @@ in {
     };
   };
 
+  vim-vint = buildPythonPackage rec {
```

Comment:
The linter looks like an executable not a python library to me. We usually do not put applications into `python-packages.nix` but into a dedicate file, which is referenced in `all-packages.nix`.